sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/TypeCoercion.scala

```diff
@@ -149,7 +149,7 @@ object TypeCoercion {
       })
   }
 
-  private def findWiderCommonType(types: Seq[DataType]): Option[DataType] = {
+  def findWiderCommonType(types: Seq[DataType]): Option[DataType] = {
```

Comment:
surprisingly, `TypeCoercion` is totally public, so this should probably be `private[sql]`.

Reply:
Actually, these were removed in [SPARK-16813](https://issues.apache.org/jira/browse/SPARK-16813) and https://github.com/apache/spark/pull/14418 :).